src/app/ipv/middleware.js

```diff
@@ -336,7 +338,10 @@ module.exports = {
         }
       }
 
-      return res.render(`ipv/page/${sanitize(pageId)}.njk`, renderOptions);
+      const sanitzedPageId = sanitize(pageId);
+
+      req.session.currentPage = sanitzedPageId;
```

Comment:
What cases is this intended to catch? Earlier in this method we have an if block that sets this to something else if `req.session.currentPage !== pageId`, so I'm not sure when it would be different.

(also typo in `sanitzed`)

Reply:
Ah this was left over from testing some changes out, I'll revert